toolkit/docs/building/dedicated_guides/build_images.md

```diff
@@ -0,0 +1,139 @@
+# --- or ---
+sudo make image CONFIG_FILE=./imageconfigs/core-efi.json REBUILD_PACKAGES=n REBUILD_TOOLS=y
+```
+
+## **I need to Validate Changes with Local Packages**
+
+The following directions will build both packages and images locally.
+
+### **1. Install Prerequisites**
+
+Prepare your system by [installing the necessary prerequisites here](../prerequisites.md).
+
+### **2. Picking a Branch or Tag**
+
+Please read [Clone and Sync to Stable Commit](../building.md#clone-and-sync-to-stable-commit) for details on picking a branch to develop from.
+
+When working with images, the `stable` approach is often the better choice but will lack bleeding edge packages. Generally when working with images, the version of a packages is less important than simply having the package available. **While much less likely with image work, still be aware that the stable branches trail behind the development branches and this can result in merge conflicts when it is time to create a PR.**
+
+The `hybrid` option minimizes the chance of package conflicts, but runs the risk of seeing unresolvable build cycles appearing.
+
+> Unlike when working on package changes, image changes are generally self-contained enough that a build with a `hybrid` or `stable` setup is sufficient for validation.
+
+### **3. Get a Toolchain**
+
+If you are using the `stable` or `hybrid` branch strategy, the toolchain will be handled automatically. Just invoke the `sudo make image ...` or `sudo make iso ...` commands as needed and the tooling will deal with the rest.
+
+If you are working on a development branch, you will need to build a toolchain. Refer to [Rebuild the Toolchain](../building.md#rebuild-the-toolchain) for details. **Save your toolchain archive for later!**
+
+### **4. Create a Custom Image Definition**
+
+See [Image Config Files](https://github.com/microsoft/CBL-MarinerTutorials/blob/main/docs/packages/working_with_packages.md#image-config-file) and [Building an Image](https://github.com/microsoft/CBL-MarinerTutorials/blob/main/docs/building/building.md) from the [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials) repo for details on working with custom images. This will cover:
+
+* Setting up an image config file
+* Adding packages to the image
+
+The image config format is [documented here](../../formats/imageconfig.md). This covers instructions on how to:
+
+* Define partition layouts
+* Configure advanced storage
+* Add users
+* Add user-defined customizations
+* Configure unattended installer behavior
+
+The core repo behaves slightly different than the dedicated build environment described in [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials). If you want to use locally built packages to build your images, use the following commands. (Otherwise please refer to [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials) for simpler methods)
```

Comment:
```suggestion
The core repo behaves slightly different than the dedicated build environment described in [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials). If you want to use locally built packages to build your images, use the following commands. Otherwise please refer to [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials). 
```